llama_stack/templates/fireworks/build.yaml

```diff
@@ -6,8 +6,6 @@ distribution_spec:
     memory:
     - meta-reference
     - remote::weaviate
-    - remote::chromadb
-    - remote::pgvector
```

Comment:
Intended?

Reply:
Yeah, we want "fireworks" / "together" distro to be remote-hosted only (and not spin up local chromadb/pgvector).